### PR TITLE
[bitnami/mariadb-galera] LDAP map and filter parameters addition

### PR DESCRIPTION
**Pull request by iomarcovalente**
**Description of the change**

This change adds MAP and FILTER search options.

**Benefits**

Having more options and granularity for non-standard LDAP searches

**Possible drawbacks**

Works only for the passwd map (posix users) - which is already the default BASE_LOOKUP map for this repo anyway.

**Applicable issues**

Related PR for upstream mariadb-galera container: https://github.com/bitnami/bitnami-docker-mariadb-galera/pull/40

**Additional information**

PLEASE NOTE ⚠️  This PR depends on changes in mariadb-galera container to be functional https://github.com/bitnami/bitnami-docker-mariadb-galera/pull/40

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb-galera/Chart.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb-galera
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 5.5.1
+version: 5.5.2
```
Comment (juan131) [inline bitnami/mariadb-galera/Chart.yaml:31]: Please bump the minor version instead since you're adding new parameters.
```suggestion
version: 5.6.0
```

**Review by juan131 — COMMENTED** [review]
Thanks for the contribution! Just a minor requirement, and we're ready to merge your changes.

File: bitnami/mariadb-galera/Chart.yaml (hunk 1 above)
Comment (iomarcovalente) [reply bitnami/mariadb-galera/Chart.yaml]: thanks for the comment - I just did bump the minor version

**Comment by iomarcovalente** [comment]
@juan131 thanks for the comment but before merging please keep in mind that the related PR on mariadb-galera container repo, as mentioned in the description, is still on-hold. Without that (bitnami/bitnami-docker-mariadb-galera#40) this change won't be fully functional.

**Comment by juan131** [comment]
@iomarcovalente new versions of the container were released adding support for this using the `LDAP_SEARCH_FILTER` and `LDAP_SEARCH_MAP` env. vars. Could you please update your PR, including these env. vars in the **statefulset.yaml**?

**Review by juan131 — APPROVED** [review]

**Comment by juan131** [comment]
Thanks for updating the PR!

**Pull request merged**